### ISSUE TITLE
Not showing sounds

### ISSUE THREAD

**Issue by alexthegamer305** [hainguyents13/mechvibes#116]
![not showing sounds](https://user-images.githubusercontent.com/88905512/129407576-ffe773ff-58c4-439f-9662-c2ff4756921c.JPG)
mechvibes is not showing any sound

i even tried redownloading it but it did not work

**Comment by ryuuzera** [comment]
same problem here

**Comment by supersealion** [comment]
From what I can tell it's due to conflicts with custom sounds. Not sure if it's having too many or if it's specific sound formats or characters in the sound file or folder names, but it's definitely a custom sound conflict for me. Trying to figure out which one by just deleting them one by one.

**Comment by Selixuz** [comment]
I had the same problem here and I checked every custom sounds that I downloaded to see the differences between them and I found out that I had downloaded 2 packs with **.ogg** and 1 that had a **.txt** config file. But I tried converting the **.ogg to .wav** but it still didn't worked, then I converted the **config.txt to config.json** and it started working again. 
But because I didn't changed the content on the .txt and just converted it, this specific sound isn't working at all but it doesn't glitch the other sounds anymore.

The .ogg files are fine, I converted them back and it still worked so for me it was just the config.txt
